### DISCUSSION TITLE
how to support libcef

### DISCUSSION THREAD

**Discussion by ifquant** [ptitSeb/box64#409] [Q&A] [answered]
i want to support cef.  can you give some advice about that.  what need to do?

**Comment by ptitSeb** [comment, marked as answer]
The issue with libcef (and everything chrome based) is libtcmalloc that is staticaly linked in it. Those libs redirect malloc/free libc operators. This needs to be done before loading any libs, but cannot really be done in box64 case because of the wrapping of the libs.

I have started working on that, with two branches: `malloc_override` and `embrace_malloc_override`, trying 2 different strategies. For now, the `malloc_override` is the most promissing, so I suggest you start here.

**Reply by ifquant** [reply]
thanks for reply. i think if we could  change some code in glibc's ld.so  ,it will be more simple? :)
